### ISSUE TITLE
Home page: When content is reflowed, the "Search" and the place holder text are not fully visible

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#141]
URL: https://rsams.dev.cloud.openplus.ca/en
https://recalls-rappels.canada.ca/en
User Agent: Firefox (v98)
Language: Both

Issue Description: When the content is reflowed, the "Search" and place holder text are not fully visible and there is overlap. Same issue applies to the production site.
![image](https://user-images.githubusercontent.com/88776404/160838482-1dfae43e-0993-4a1c-a791-634e72d49e92.png)

Steps to reproduce:

1. Navigate to the url above. Or Page name for the test is: 'Find recalls, advisories and safety alerts' in case it needs to be searched on for next test.
2. Set the desktop browser to 1280(W) X 1024(H) and zoom in to 400% (Firefox -> Web Developer settings ->Resize Window to 1280X1024).

Expected results: When content is zoomed up to 400% the content should be presented without loss of information or text overlap.

**Comment by echouin** [comment]
@XiuJuanXia From OP: we hide the text on mobile so you get only the magnifying glass.
﻿
﻿code would look like this
﻿
﻿﻿<button class="btn btn-default button js-form-submit form-submit icon-before" data-drupal-selector="edit-submit" type="submit" id="edit-submit" name="op" value="Recherche">
<span class="icon glyphicon glyphicon-search" aria-hidden="true"></span>

<span class="sr-only">Recherche</span>

</button>

﻿

﻿visually that would look like the attached
![image](https://user-images.githubusercontent.com/88099454/161834329-2673ded7-492b-42f6-9a34-d9eb75cbd296.png)

**Comment by XiuJuanXia** [comment]
@echouin Can this be verified in dev or prod? I don't see the change in dev or prod.
This is what we see in PROD in iphone X:
![image](https://user-images.githubusercontent.com/88776404/161973322-e9e5a03e-e168-4681-a8a8-ec414be6cd5c.png)
![image](https://user-images.githubusercontent.com/88776404/161973341-56be2e97-cd64-4742-b992-fd57b64a3f0b.png)

**Comment by echouin** [comment]
@XiuJuanXia OP was asking us to check w/ you if that would solve the issue. Once I have your confirmation, they will initiate.

Will the above-noted solution solve the issue?

**Comment by XiuJuanXia** [comment]
@echouin, the placeholder text (Rechercher des alertes et des rappels) should be fully visible in a mobile view.

**Comment by zimw7** [comment]
The issue has been resolved. The placeholder text has been removed, and the search field displays correctly at 400% zoom.
![image](https://github.com/user-attachments/assets/cd67e518-f8f4-4102-8adc-974d27176bf6)

**Comment by 40999221** [comment]
Issue resolved. Placeholder text removed and zoom at 400% doesn't missed any content of the page.

![image](https://github.com/user-attachments/assets/2dc8f81d-a489-405e-a187-01bdf490ed85)

**Comment by YuxuanXie01** [comment]
Retested, issue fixed. 
![image](https://github.com/user-attachments/assets/1bd430f6-2aa0-48ee-8cbd-dc1c4c8430c0)
